### ISSUE TITLE
How to forecast time series data using Neural Network toolbox in Matlab

### ISSUE THREAD

**Issue by linetachieng** [section-engineering-education/engineering-education#6910]
### Topic Suggestion
How to forecast time series data using Neural Network toolbox in Matlab

### Writing sample(s): 
https://www.section.io/engineering-education/matlab-graphical-user-interface/

#### Introduction
Forecasting refers to predicting what will happen in the future by taking into consideration events in the past and present. Time series data is a collection of observations obtained through repeated measurements. A neural network toolbox is a Matlab toolbox that helps implement neural networks without writing any code. Using this toolbox, we can forecast time series data easily.
This tutorial will look at the general view of neural networks. Also, we will see how to use this toolbox to implement the neural network. Finally, we will see how to develop a model and generate the code for that model. You can use this code elsewhere for various forecasting purposes.

#### Key takeaways 
- How to use Neural Network toolbox.
- Understanding of the neural networks.
- How to import your dataset into the toolbox.
- How to generate the code for the created model.

#### Article quality
This article covers the neural network toolbox into deep and its various properties.

### Templates to use as guides
- [How To Guide Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md)
- [Software Review Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md)
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)

**Comment by ahmadmardeni1** [comment]
How does your article differ from https://www.google.com/search?q=How+to+forecast+time+series+data+using+Neural+Network+toolbox+in+Matlab&oq=How+to+forecast+time+series+data+using+Neural+Network+toolbox+in+Matlab&aqs=edge..69i57j69i61.817j0j1&sourceid=chrome&ie=UTF-8 ? @linetachieng

**Comment by linetachieng** [comment]
@ahmadmardeni1 Here, we use the neural toolbox in Matlab to train the network. From this toolbox, we generate the code neural network that we have created. This generated code can be implemented for various applications for testing and use.

**Comment by ahmadmardeni1** [comment]
Yes that's what other tutorials are also doing. What makes your article unique/different? @linetachieng

**Comment by linetachieng** [comment]
@ahmadmardeni1, if you look at the articles in the link you sent before, the tutorials shows how you can create a neural network by code. In my article, we bring the concept of using the neural network toolbox to create these networks using Matlab's available databases.

**Comment by ahmadmardeni1** [comment]
Okay please make sure your article does not overlap with any of them. Approved. @linetachieng

**Comment by WanjaMIKE** [comment]
Closed #6910 via #7447
